data/transactions/logic/eval.go

```diff
@@ -5828,6 +5828,16 @@ func opBlock(cx *EvalContext) error {
 		cx.Stack[last] = stackValue{Uint: hdr.Bonus.Raw}
 	case BlkProposerPayout:
 		cx.Stack[last] = stackValue{Uint: hdr.ProposerPayout.Raw}
+
+	case BlkBranch512:
```

Comment:
maybe we've already discussed this but do we have any opcodes that modify bytes in-place?